docs/css/aspect-ratio.md

```diff
@@ -0,0 +1,72 @@
+> Layout
+
+# Aspect Ratio
+
+Utilities for controlling the aspect ratio of an element.
+
+## Quick reference
+
+::: warning Warning
+  Known problem: the shortcuts `aspect-ratio`, `aspect-square`, `aspect-video` are not working as intended.
+:::
```

Comment:
Just checking, is this still valid @Skadefryd?

Reply:
Still applies, we are still on the margin-trick for this , and those are part of the css native aspect-ratio .  (Does not interchange because the margin trick way of doing this needs to be set on a parent while aspect-ratio as  css is set directly on the thing.